content/en/apps/reference/app-settings/transitions.md

```diff
@@ -381,6 +406,23 @@ Entries in `muting_history` contain the following information:
 | date | Date in ISO Format |
 | report_id | An `_id` reference to the report that triggered the action |
 
+
+##### [11] Client-side Muting history as of 3.12.0
```

Comment:
worth noting in contacts schema page?
https://docs.communityhealthtoolkit.org/core/overview/db-schema/#contacts-persons-and-places

Reply:
Unsure. This is an implementation detail, I wouldn't want it: a. edited or b. used for any other purpose by data-analysis and such.

Reply:
it is in the contact doc, right? it is going to get used :( 
if that's not a good idea - maybe that's what you should say in the schema documentation?

Reply:
It is in the contact doc, correct. We need this metadata to insure some sort of consistency with out of order syncing and muting. The alternative would have been to have another doc type to sync to just contain this metadata.

Reply:
Server-side Muting history exists in infodocs, and can be used for dashboards.

Reply:
> The alternative would have been to have another doc type to sync to just contain this metadata

Sounds not that nice.

I'm not really saying you shouldn't put the data here. But I'm saying that this data is very public and discoverable, so it is likely it will be used.

Reply:
I can't guarantee that this data structure won't change over time. It being an implementation artifact, I don't want to block future development from changing this structure to accommodate new functionalities or bug fixes because data analysis is already using it. 
How do you think it's best communicated that this data is not guaranteed to have this format or exist at all?

Reply:
most people writing data analytic queries are doing it from xform data. with that data, all the internal calculations/intermediate steps/UI strings are even exposed publicly. it is very common for a query to depend on something fragile which either a) breaks the query, or b) makes it more difficult to edit the form later.

we would benefit from a convention of "private vs public data", but i believe we'd be creating it here. today, everything is fair game cc @derickl